theme_paptic/data/ir_asset.xml

```diff
@@ -1,21 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data>
+    <record id="theme_paptic.primary_variables_scss" model="theme.ir.asset">
+        <field name="key">theme_paptic.primary_variables_scss</field>
+        <field name="name">Primary variables SCSS</field>
+        <field name="bundle">web._assets_primary_variables</field>
+        <field name="path">theme_paptic/static/src/scss/primary_variables.scss</field>
+    </record>
 
-        <record id="theme_paptic.primary_variables_scss" model="theme.ir.asset">
-            <field name="key">theme_paptic.primary_variables_scss</field>
-            <field name="name">Primary variables SCSS</field>
-            <field name="bundle">web._assets_primary_variables</field>
-            <field name="path">theme_paptic/static/src/scss/primary_variables.scss</field>
-        </record>
-
-        <record id="theme_paptic.bootstrap_overridden_scss" model="theme.ir.asset">
-            <field name="key">theme_paptic.bootstrap_overridden_scss</field>
-            <field name="name">Bootstrap overridden SCSS</field>
-            <field name="bundle">web._assets_frontend_helpers</field>
-            <field name="directive">prepend</field>
-            <field name="path">theme_paptic/static/src/scss/bootstrap_overridden.scss</field>
-        </record>
-
-    </data>
+    <record id="theme_paptic.bootstrap_overridden_scss" model="theme.ir.asset">
+        <field name="key">theme_paptic.bootstrap_overridden_scss</field>
+        <field name="name">Bootstrap overridden SCSS</field>
+        <field name="bundle">web._assets_frontend_helpers</field>
+        <field name="directive">prepend</field>
+        <field name="path">theme_paptic/static/src/scss/bootstrap_overridden.scss</field>
+    </record>
```

Comment:
When you reindent, reindent following our convention, no indentation and blank line around:

```xml
<odoo>

<record>
[...]
</record>

</odoo>
```